src/deck-options.md

```diff
@@ -4,260 +4,221 @@
+- **Rename**: Changes the name of the current preset.
+- **Delete**: Deletes the current preset. This will make your next sync
   a one-way sync.
-- **Save to all subdecks**. Like _Save_, but also assigns the selected preset to all
+- **Save to all subdecks**: Like "Save", but also assigns the selected preset to all
   subdecks of the currently selected deck.
-- **Optimize all presets**. When FSRS is enabled, this allows you to optimize the parameters of all presets at once.
+- **Optimize all presets**: When FSRS is enabled, this allows you to optimize the parameters of all presets at once.
 
-Deck Options are not retroactive. For example, if you change an option that
+Deck options are not retroactive. For example, if you change an option that
 controls the delay after failing a card, cards that you failed before 
-changing the option will have the old delay, not the new one.
+changing this option will still have the old delay, not the new one.
 
 ## Subdecks
 
-If your deck has subdecks, each deck can optionally be assigned a different preset.
+If your deck has subdecks, you can assign each subdeck a different preset.
 When Anki shows a card, it will check which subdeck the card is in, and use the options
 for that deck. There are some exceptions:
 
-- The [display order](#display-order) options in the v3 scheduler are taken from the
+
+- The "New cards/day" and "Maximum reviews/day" [limits](#daily-limits) of a subdeck affects the number of cards gathered from that subdeck. But the amount of cards you see depends on the limits set on the deck you select to study.
+- The [display order](#display-order) options in the are taken from the
   deck you select to study, not the deck of the current card.
 
 For example, let's say you have this collection:
 
     - Deck A (Preset 1)
       - Deck A::Subdeck B (Preset 2)
-        - Card B1
-        - Card B2
 
-Presets 1 and 2 are identical, with two exceptions:
+Preset 1 and Preset 2 are identical, with two exceptions:
 
 - Preset 1:
-- New Cards - Learning steps: 1m 10m
-- Display Order - New/review priority: Mix with reviews
+  - Learning steps: 1m 10m
+  - New/review order: Mix with reviews
 - Preset 2:
-- New Cards - Learning steps: 20m 2h
-- Display Order - New/review priority: Show after reviews
+  - Learning steps: 20m 2h
+  - New/review order: Show after reviews
+
+If you choose to study Subdeck B:
+
+
+- Learning steps for all new cards will be 20m 2h (Preset 2 applies).
+- All new cards will be shown after reviews (Preset 2 applies).
+
+
+
 
 If you choose to study Deck A:
 
-- All new cards will be mixed with reviews (preset 1 applies)
+- Learning steps for new cards in Deck A will be 1m 10m (Preset 1 applies).
+- Learning steps for new cards in Subdeck B will be 20m 2h (Preset 2 applies).
+- All new cards will be mixed with reviews (Preset 1 applies).
 
-If you choose to study Subdeck B:
 
-- All new cards will be shown after reviews (preset 2 applies)
```

Comment:
```suggestion
    - Deck A (Preset 1)
      - New Card A
      - Deck A::Subdeck B (Preset 2)
        - New Card B

Preset 1 and Preset 2 are identical, with two exceptions:

- Preset 1:
  - Learning steps: 1m 10m
  - New/review order: Mix with reviews
- Preset 2:
  - Learning steps: 20m 2h
  - New/review order: Show after reviews

If you select Deck A to study:
- Learning steps for New Card A will be 1m 10m (Preset 1 applies).
- Learning steps for New Card B will be 20m 2h (Preset 2 applies).
- New cards will be mixed with reviews (Preset 1 applies).

If you select Subdeck B to study:
- Learning steps for New Card B will be 20m 2h (Preset 2 applies).
- New cards will be shown after reviews (Preset 2 applies).
```
I tried to reformulate this working backward from what we want the example to show, and I think this is closer to that goal.

I assume that you switched A and B to make the juxtaposition more clear, but I think it is more confusing to take them out of order.

Reply:
Not resolved. 

> You're introducing extra elements here

The original example always had 2 cards, so they aren't extra elements. I encourage you to approach it from the end result as well. What should this example demonstrate for the reader? From the way it is written, it seems that it needs to show something that will be controlled by the click-to-study deck, and something that won't. If you have a different understanding of the purpose of the example, let's talk about that. 

> "Deck A::Subdeck B" also doesn't help.

I didn't change that. That's the currently published text, and part of your PR.